source/stylesheets/refills/_accordion.scss

```diff
@@ -1,54 +1,32 @@
 .accordion {  
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-border-color: gainsboro !default;
-  $base-line-height: 1.5em !default;
-  $base-spacing: 1.5em !default;
-  $dark-gray: #333 !default;
-  $base-font-color: $dark-gray !default;
-  
-  margin: 0;
-  padding: 0;
-
-  ul {
-    margin: 0;
-    padding: 0;
-  }
-
-  li {
-    list-style: none;
-  }
-
-  p {
-    color: $base-font-color;
-    line-height: $base-line-height;
-  }
-
-  a {
-    text-decoration: none;
-  }
-  //////////////////////////////////////////////////////////////////////////////////
-
-  $accordion-menu-border-color: $base-border-color;
-  $accordion-menu-color: $base-font-color;
-  $accordion-menu-border: 1px solid $accordion-menu-border-color;
-  $accordion-menu-background: lighten($accordion-menu-border-color, 10%);
+  $accordion-menu-color: $base-font-color-dark;
+  $accordion-menu-border: $base-border;
+  $accordion-menu-background: $base-box-color;
   $accordion-menu-hover: lighten($accordion-menu-background, 2%);
   $accordion-menu-sub-background: darken($accordion-menu-background, 5%);
   $accordion-menu-sub-inset: darken($accordion-menu-sub-background, 6%);
   $accordion-menu-sub-hover: lighten($accordion-menu-sub-background, 2%);
-  $accordion-menu-list-padding: ($base-spacing / 2) $gutter;
+  $accordion-menu-list-padding: $small-spacing $gutter;
   
   background-color: $accordion-menu-background;
   border: $accordion-menu-border;
-  margin-bottom: $base-spacing;
+  margin: 0 0 $base-spacing 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 $base-spacing` instead of `0 0 $base-spacing 0`